test/uk/gov/hmrc/gatekeeper/mocks/services/DeveloperServiceMockProvider.scala

```diff
@@ -70,8 +80,10 @@ trait DeveloperServiceMockProvider {
     }
 
     object SeekRegisteredUser {
-      def returnsFor(email: String, verified: Boolean = true, mfaEnabled: Boolean = true) =
-        when(mockDeveloperService.seekUser(eqTo(email))(*)).thenReturn(successful(Some(RegisteredUser(email, idOf(email), "first", "last", verified = verified, mfaEnabled = mfaEnabled))))
+      def returnsFor(email: String, verified: Boolean = true, mfaEnabled: Boolean = true) = {
+
+        when(mockDeveloperService.seekUser(eqTo(email))(*)).thenReturn(successful(Some(RegisteredUser(email, idOf(email), "first", "last", verified = verified, mfaEnabled = mfaEnabled, mfaDetails = mfaEnabledToMfaDetails(mfaEnabled)))))
```

Comment:
Could we use the mock module for mocking out seekUser?
https://github.com/hmrc/api-gatekeeper-frontend/blob/main/test/uk/gov/hmrc/gatekeeper/mocks/services/DeveloperServiceMockProvider.scala